### PR TITLE
Adding OCP 4.11 GA config files

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.11-ga"
+  terraform_version: "0.12.26"
+  ignition_version: "3.2.0"
+  ocp_channel: "fast"
+  ocp_version_index: -2
+ENV_DATA:
+  vm_template: 'rhcos-4.10.3-x86_64-vmware.x86_64'
+  acm_hub_channel: release-2.4
```
[the remark this answers is not in the document]
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml:16]: ack. I will change in next commit with other changes

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.11-ga"
+  terraform_version: "0.12.26"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.10/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: "rhcos-4.10.3-x86_64-vmware.x86_64"
+  acm_hub_channel: release-2.4
```
[the remark this answers is not in the document]
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml:17]: ack. I will change in next commit with other changes

[the remark this answers is not in the document]
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml]: done

File: ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-minus1-config.yaml]: done

**Review by petr-balogh — APPROVED** [review]

**Review by prsurve — APPROVED** [review]

**Pull request merged**